modules/installation-generate-ignition-configs.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// Consider also adding the installation-configuration-parameters.adoc module.
+
+[id="installation-generate-ignition-configs-{context}"]
+= Creating the ignition config files
+
+Because you need to manually start the cluster machines, you must generate the
+ignition config files that the cluster needs to make its machines.
+
+.Prerequisites
+
+* Obtain the {product-title} installation program and the access token for your
+cluster.
+* Generate the `install-config.yaml` file and complete any modifications
+to it.
+
+.Procedure
+
+* Obtain the ignition configuration files:
++
+----
+$ ./openshift-install create ignition-configs --dir=<installation-directory> <1>
+----
+<1> Specify the directory name that contains the `install-config.yaml` file for
+your cluster.
++
+The `install-config.yaml` file is removed, and the following files are generated
+in the directory:
++
+----
+.
+├── auth
```

Comment:
changes in flight https://github.com/openshift/installer/pull/1592#issuecomment-481886812